linodecli/configuration/helpers.py

```diff
@@ -124,63 +124,69 @@ def _handle_no_default_user(self):
     """
     users = [c for c in self.config.sections() if c != "DEFAULT"]
 
-    if len(users) == 1:
```

Comment:
Refactor code and use case switch to fix a lint issue: `Too many branches`

Reply:
@yec-akamai `match ... case ...` is the new featured added to Python 3.10. Since we want to maintain backward compatibility to 3.7 or above, can you consider use `if ... else ...` statements here and use another method to get rid of the lint warning, or if it is impossible to avoid, you can disable the lint for this type of warning.

Reply:
I see. I'll try to figure out another way, but probably have to disable the lint eventually. Thanks for the heads up!